src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -46,8 +76,119 @@ export default class GroupComparisonMutationMapper extends MutationMapper<
+                genomeNexusCache={this.props.genomeNexusCache}
+                genomeNexusMutationAssessorCache={
+                    this.props.genomeNexusMutationAssessorCache
+                }
+                dataStore={
+                    this.props.store.dataStore as MutationMapperDataStore
+                }
+                downloadDataFetcher={this.props.store.downloadDataFetcher}
+                myCancerGenomeData={this.props.store.myCancerGenomeData}
+                hotspotData={this.props.store.indexedHotspotData}
+                indexedVariantAnnotations={
+                    this.props.store.indexedVariantAnnotations
+                }
+                indexedMyVariantInfoAnnotations={
+                    this.props.store.indexedMyVariantInfoAnnotations
+                }
+                oncoKbData={this.props.store.oncoKbData}
+                usingPublicOncoKbInstance={
+                    this.props.store.usingPublicOncoKbInstance
+                }
+                mergeOncoKbIcons={this.props.mergeOncoKbIcons}
+                onOncoKbIconToggle={this.props.onOncoKbIconToggle}
+                civicGenes={this.props.store.civicGenes}
+                civicVariants={this.props.store.civicVariants}
+                enableOncoKb={this.props.enableOncoKb}
+                enableFunctionalImpact={this.props.enableGenomeNexus}
+                enableHotspot={this.props.enableHotspot}
+                enableMyCancerGenome={this.props.enableMyCancerGenome}
+                enableCivic={this.props.enableCivic}
+                generateGenomeNexusHgvsgUrl={
+                    this.props.generateGenomeNexusHgvsgUrl
+                }
+                isCanonicalTranscript={this.props.store.isCanonicalTranscript}
+                selectedTranscriptId={this.props.store.activeTranscript.result}
+                columnVisibility={this.props.columnVisibility}
+                storeColumnVisibility={this.props.storeColumnVisibility}
+                namespaceColumns={this.props.store.namespaceColumnConfig}
+                columns={this.columns}
+                profiledPatientCountsByGroup={
+                    this.props.groupToProfiledPatientCounts
+                }
+                groups={this.props.groups}
+                formatPaginationStatusText={this.formatPaginationStatusText}
+                showTotalMutationCountsInCountHeader={true}
+                sampleSet={this.props.sampleSet}
+                customControls={this.tableCustomControls}
+                getRowDataByProteinChange={
+                    this.rowDataByProteinChangeForEnrichedGroups
+                }
+                initialSortColumn={'p-Value'}
+                initialSortDirection={'asc'}
+            />
+        );
+    }
+
+    @computed get rowDataByProteinChange() {
+        let dataStore = this.props.store.dataStore as MutationMapperDataStore;
+        let mutationsByProteinChange = _.values(
```

Comment:
you should avoid nesting lodash.  use chaining instead. 
also, this seems strange because, unless i'm mistaken, will result in array of arrays?  where mutationsByPorteinChange suggests a map of proteinChange to mutations.